scenes/templates/scenes/playlists/create_inner.html

```diff
@@ -0,0 +1,12 @@
+{% load bulma_tags %}
+<div class="box">
+    <div class="columns">
+        <div class="column">
+            <form method="POST">
+                {% csrf_token %}
+                {{ form|bulma }}
+                <button type="submit" class="button is-primary" id="playlist-add-submit">Create Playlist</button>
```

Comment:
Whoops, that's from an artifact left by me in the first implementation. We don't actually need that id here
```suggestion
                <button type="submit" class="button is-primary">Create Playlist</button>
```